src/EditorFeatures/Core/SymbolSearch/SymbolSource.cs

```diff
@@ -0,0 +1,71 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.Editor.FindUsages;
+using Microsoft.CodeAnalysis.Editor.GoToBase;
+using Microsoft.CodeAnalysis.Shared.Extensions;
+using Microsoft.CodeAnalysis.Text;
+using Microsoft.VisualStudio.Core.Imaging;
+using Microsoft.VisualStudio.Language.Intellisense.SymbolSearch;
+
+namespace Microsoft.CodeAnalysis.Editor.SymbolSearch
+{
+    class SymbolSource : ISymbolSourceFromLocation
```

Comment:
1. internal.
2. class name should match interface name.

Reply:
this class will implement more interfaces "ISymbolSource..." in the future